distributed-commandbus-springcloud/src/main/java/org/axonframework/springcloud/commandhandling/SpringCloudHttpBackupCommandRouter.java

```diff
@@ -266,4 +176,108 @@ private static URI buildURIForPath(URI uri, String appendToPath) {
+            return this;
+        }
+
+        @Override
+        public Builder routingStrategy(RoutingStrategy routingStrategy) {
+            super.routingStrategy(routingStrategy);
+            return this;
+        }
+
+        @Override
+        public Builder serviceInstanceFilter(
+                Predicate<ServiceInstance> serviceInstanceFilter) {
+            super.serviceInstanceFilter(serviceInstanceFilter);
+            return this;
+        }
+
+        @Override
+        public Builder consistentHashChangeListener(ConsistentHashChangeListener consistentHashChangeListener) {
+            super.consistentHashChangeListener(consistentHashChangeListener);
+            return this;
+        }
+
+        /**
+         * Sets the {@link RestTemplate} used as the backup mechanism to request another member's
+         * {@link MessageRoutingInformation} with.
+         *
+         * @param restTemplate the {@link RestTemplate} used as the backup mechanism to request another member's
+         *                     {@link MessageRoutingInformation} with.
+         * @return the current Builder instance, for a fluent interfacing
+         */
+        public Builder restTemplate(RestTemplate restTemplate) {
+            assertThat(restTemplate,
+                       Objects::nonNull,
+                       () -> new AxonConfigurationException("RestTemplate may not be null"));
+            this.restTemplate = restTemplate;
+            return this;
+        }
+
+        /**
+         * Sets the {@code messageRoutingInformationEndpoint} of type {@link String}, which is the endpoint where to
+         * retrieve the another nodes message routing information from. Defaults to endpoint
+         * {@code "/message-routing-information"}.
+         *
+         * @param messageRoutingInformationEndpoint the endpoint where to retrieve the another nodes message routing
+         *                                          information from
+         * @return the current Builder instance, for a fluent interfacing
+         */
+        public Builder messageRoutingInformationEndpoint(String messageRoutingInformationEndpoint) {
+            assertThat(messageRoutingInformationEndpoint,
+                       name -> Objects.nonNull(name) && !name.equals(""),
+                       () -> new AxonConfigurationException("The messageRoutingInformationEndpoint may not be null"));
+            this.messageRoutingInformationEndpoint = messageRoutingInformationEndpoint;
+            return this;
+        }
+
+        /**
+         * Initializes a {@link SpringCloudHttpBackupCommandRouter} as specified through this Builder.
+         *
+         * @return a {@link SpringCloudHttpBackupCommandRouter} as specified through this Builder
+         */
+        public SpringCloudHttpBackupCommandRouter build() {
+            return new SpringCloudHttpBackupCommandRouter(this);
+        }
+
+        private void validate() {
+            assertThat(restTemplate,
```

Comment:
Extraction to a separate method.

Reply:
Replaced for the `BuilderUtils` functions.
What do you think?